envoy/config/filter/accesslog/v2/accesslog.proto

```diff
@@ -358,16 +361,32 @@ message NotHealthCheckFilter {
 message TraceableFilter {
 }
 
-// Filters for random sampling of requests. Sampling pivots on the header
-// :ref:`x-request-id<config_http_conn_man_headers_x-request-id>` being present. If
-// :ref:`x-request-id<config_http_conn_man_headers_x-request-id>` is present, the filter will
-// consistently sample across multiple hosts based on the runtime key value and the value extracted
-// from :ref:`x-request-id<config_http_conn_man_headers_x-request-id>`. If it is missing, the
-// filter will randomly sample based on the runtime key value.
+// Filters for random sampling of requests.
 message RuntimeFilter {
-  // Runtime key to get the percentage of requests to be sampled. This runtime
-  // control is specified in the range 0-100 and defaults to 0.
+  // Runtime key to get the percentage of requests to be sampled.
   string runtime_key = 1 [(validate.rules).string.min_bytes = 1];
+
+  // The default sampling rate if a value is not found in runtime. Defaults to 0.
+  uint32 default = 2;
```

Comment:
why uint32 here but google's UInt32 below?

Reply:
because the default is 0.

Reply:
I think `default_sample_rate` would be more descriptive. Also, it's not really a rate unless there is a divisor, so maybe point out it's divided by divisor below. Should we use `core.Percent`? and `divisor` as the granularity? Not sure.

Reply:
Happy to change the name of the field. The reason I didn't do `core.Percent` is IMO it's pretty strange to use that field and that tack on a divisor. This goes back to our epic thread about whether we want a `RoundedPercent` message or something like that. IMO we do want that and that would make a case like this much more clear. LMK what you think and I can update ... at some point. :)

Reply:
Yeah, if we're going to do this in more than one place in the API, then we would benefit from something like `RoundedPercent`.